sklearn/utils/estimator_checks.py

```diff
@@ -2501,3 +2514,42 @@ def check_fit_idempotent(name, estimator_orig):
         if hasattr(estimator, method):
             new_result = getattr(estimator, method)(X_test)
             assert_allclose_dense_sparse(result[method], new_result)
+
+
+def check_outlier_rejectors(name, estimator_orig):
```

Comment:
Not sure if we really need this in common tests...?

Reply:
This is an artifact from when this PR was only about outlier rejectors. Where should this be moved?

Reply:
I'm not sure since the outlier rejectors are spread out across multiple sections of sklearn